### ISSUE TITLE
Add import support

### ISSUE THREAD

**Issue by yarikoptic** [DanielDent/git-annex-remote-rclone#32]
A complement to #30 
There is WiP by @joeyh within git-annex to interface ability to import from external special remotes. See https://git-annex.branchable.com/design/external_special_remote_protocol/export_and_import_appendix/

**Comment by joeyh** [comment]
No code yet, just a draft for the protocol.
I think I'll probably want to get back to that in not too long.
(Globus may also want to use that.)

-- 
see shy jo

**Comment by yarikoptic** [comment]
If I read the page/history of that page on git annex website, import is supported now, right @joeyh ?

**Comment by Lykos153** [comment]
No, not for external remotes. Only for (some?) internal ones, like S3 and directory. The page still states
`(This part is a draft, not implemented yet.)` and there hasn't been any release note saying otherwise. (I follow them quite thoroughly because of this particular feature ;) )
